src/components/MainComposer/MainComposer.module.scss

```
@@ -88,3 +88,10 @@
 
   margin-bottom: 2.4rem;
 }
+
+.contentBotBotInputMedia {
```

Quer explicar o nome dessa classe? Nao ficou boa nao. BotBotInputMedia? Esquisito né, vale mudar.

```suggestion
.contentBotBotInputMedia {
```